docker-compose-with-temporal.yml

```diff
@@ -130,6 +135,45 @@ services:
     privileged: true
     restart: on-failure
 
+  telemetry:
+    image: "telemetry:ra-dev-0" # TODO: change to public image
+    restart: on-failure
+    env_file: docker.env
+    ports:
+      - '9125:9125/udp'
+    command:
+      - bash
+      - -c
+      - |
+        if [[ "$${RTEL_ENABLED:-false}" == "false" ]]; then
+          echo 1>&2 "RTEL_ENABLED is not true, going to sleep."
+          sleep inf
```

Comment:
why not gracefully shut down at this point?

Reply:
because I need `restart: on-failure` for the enabled case, but if the container exits when disabled (default) it will restart loop :/